dev-python/tlsh/tlsh-4.8.2.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..9} )
+inherit distutils-r1
+
+DESCRIPTION="Fuzzy matching library - C++ extension for Python"
+HOMEPAGE="https://pypi.org/project/python-tlsh/"
+SRC_URI="https://github.com/trendmicro/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S=${WORKDIR}/${P}/py_ext
+
+LICENSE="|| ( Apache-2.0 BSD )"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	dev-libs/tlsh
```

Comment:
Is it statically linked or something? (DEPEND only)?

Reply:
No, ldd tells me this:

```
$ ldd /usr/lib64/libtlsh.so
	linux-vdso.so.1 (0x00007fffefb15000)
	libstdc++.so.6 => /usr/lib/gcc/x86_64-pc-linux-gnu/12.0.1/libstdc++.so.6 (0x00007f621c28d000)
	libgcc_s.so.1 => /usr/lib/gcc/x86_64-pc-linux-gnu/12.0.1/libgcc_s.so.1 (0x00007f621c26d000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f621c076000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f621bf9f000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f621c4e0000)
```

Reply:
But dev-python/tlsh surely needs the library at runtime too?

Reply:
should be fixed now